src/Providers/Http/Util/ErrorMessageExtractor.php

```diff
@@ -33,6 +34,18 @@ public static function extractFromResponseData($data): ?string
             return null;
         }
 
+        // Handle [ { "error": { "message": "Error text" } } ]
+        if (
+            isset($data[0]) &&
+            is_array($data[0]) &&
+            isset($data[0]['error']) &&
+            is_array($data[0]['error']) &&
+            isset($data[0]['error']['message']) &&
+            is_string($data[0]['error']['message'])
+        ) {
```

Comment:
You don't need to check each layer. PHP supports checking multiple layers down. 👍 
```suggestion
        if (isset($data[0]['error']['message']) && is_string($data[0]['error']['message'])) {
```

Reply:
But PHPStan hates it 😢